requirements.txt

```diff
@@ -1,3 +1,3 @@
-scipy
-torch
-torchvision
+scipy==1.9.*
+torch==1.12.*
+torchvision==0.13.*
```

Comment:
I normally pin dependencies to avoid issues when they release a new major or minor version that may break backward-compatibility, and also to indicate to users what version they are expected to have.

I used the latest versions from PYPI, without the bugfix version part.

Reply:
Maybe worth adding streamflow==0.2.0.dev1 too?

Reply:
> I normally pin dependencies to avoid issues when they release a new major or minor version that may break backward-compatibility, and also to indicate to users what version they are expected to have.

This is a good idea.

> I used the latest versions from PYPI, without the bugfix version part.

I'll search for the exact versions used for the paper experiments and I'll put them.

> Maybe worth adding streamflow==0.2.0.dev1 too?

It is better to leave `streamflow` separated because these dependencies here must be installed on all the nodes (aggregator and workers), while StreamFlow must be installed only on the control node (which can be the aggregator node itself, or even a different node).

Reply:
> I'll search for the exact versions used for the paper experiments and I'll put them.
The versions you already put are correct, we used the last ones.